### ISSUE TITLE
Integration Functionality

### ISSUE THREAD

**Issue by quintrino** [beeminder-capstone/Nectar-Backend#3]
Hi, 

Nectar looks great, but where do you actually list what are the features of each of the integrations? Do I have to authorize them before I know what i can beemind?

**Comment by tdulcet** [comment]
@quintrino No, you can go to `/credentials/[provider]` before completing OAuth with a provider to see the list of metrics. `[provider]` is from this line: https://github.com/beeminder-capstone/Nectar-Backend/blob/develop/config/initializers/providers.rb#L9.

For example, if you wanted to see the metrics for the Facebook integration before completing OAuth, you could go to: https://beemindernectar.herokuapp.com/credentials/facebook.

**Comment by quintrino** [comment]
Ah okay, I was mainly interested to see what the Wunderlist, Drive and Slack integrations were but it looks like they aren't running yet, but exciting to know they're on the roadmap for the future!
